Backend/NoCode.FlowerShop.Domain/Customer.cs

```diff
@@ -8,6 +9,8 @@ public sealed class Customer : Entity<Guid>
     public string Password { get; private set; }
     public byte[] Salt { get; }
 
+    [Timestamp]
+    public byte[]? Version { get; private set; }
```

Comment:
Can add new line after this (super optional)

Reply:
will do in next pr.